scene/gui/graph_edit.cpp

```diff
@@ -1169,6 +1176,10 @@ HBoxContainer *GraphEdit::get_zoom_hbox() {
 	return zoom_hb;
 }
 
+void GraphEdit::_visual_script_request(const String &p_text, const String &p_category) {
```

Comment:
This doesn't seem to be called anywhere?

Reply:
You do connect to the signal in `modules/visual_script/visual_script_editor.cpp`, but as far as I can tell it will never be emitted.